ee/desktop/runner/runner.go

```diff
@@ -150,7 +150,7 @@ func New(opts ...desktopUsersProcessesRunnerOption) *DesktopUsersProcessesRunner
 		updateInterval:         time.Second * 5,
 		procsWg:                &sync.WaitGroup{},
 		interruptTimeout:       time.Second * 10,
-		usersFilesRoot:         filepath.Join(os.TempDir(), "kolide-desktop"),
+		usersFilesRoot:         agent.TempPath("kolide-desktop"),
```

Comment:
Ideally, we should create this temp directory and ensure proper permissions, instead of only defining a path.

Reply:
This one I think we need to think harder about. But this is clearly bug-compatible, so we could take this as improvement and iterate.

Reply:
Maybe we make `usersFilesRoot` a required param to avoid the temp path all together. Permissions aside.